lib/github_fetcher.rb

```diff
@@ -12,13 +12,15 @@ def initialize(team)
     @exclude_titles = team.exclude_titles.map(&:downcase).uniq
     @labels = {}
     @repos = team.repos
+    @dependabot_prs_only = dependabot_prs_only
   end
 
   def list_pull_requests
     pull_requests_from_github
       .reject { |pr| hidden?(pr) }
       .map { |pr| present_pull_request(pr) }
       .sort_by { |pr| pr[:date] }.reverse
+      .filter { |pr| @dependabot_prs_only == pr[:author]&.include?("dependabot") }
```

Comment:
Nitpick: Have we come across any PRs where `author` isn't specified? Defensive code is good but it's added complexity if not needed (i.e. does removing the `&` break any tests or runtime behaviour?).

Reply:
I think this is leftover from some iteration of this that didn't quite work. Will remove.